web/themes/new_weather_theme/templates/partials/radar.html.twig

```diff
@@ -23,5 +26,60 @@
          <div id="wx_radar_container" style="height: 300px;"></div>
       </wx-radar>
     </div>
+
+    {% set intensities = [{
+      dbz: "−35–0",
+      description: "Extremely light (drizzle/snow)",
+      gradient: "180deg, #8E827E 0%, #969155 17.64%, #A5A36D 27.53%, #D0D2B2 54.47%, #7E8BAF 100%"
+    },{
+      dbz: "0–20",
+      description: "Very light precipitation or general clutter",
+      gradient: "180deg, #7B88AE 0%, #5C71A6 17.64%, #445FA0 36.39%, #5DA9CC 61.25%, #59C0BA 73.2%, #52D6A2 87.85%, #3FD657 100%"
+    },{
+      dbz: "20–40",
+      description: "Light precipitation",
+      gradient: "180deg, #3FD657 0%, #3ED624 9.47%, #24890E 36.39%, #176108 61.25%, #819F06 73.2%, #FBE000 85.5%, #F4CB17 100%"
+    },{
+      dbz: "40–50",
+      description: "Moderate precipitation",
+      gradient: "180deg, #F4CB17 0%, #EBB32D 25.38%, #F9B103 72.13%, #F52D04 73.91%, #D10808 99.72%"
+    },{
+      dbz: "50–65",
+      description: "Heavy precipitation or some hail",
+      gradient: "180deg, #D10808 0%, #A20F10 16.84%, #B00301 48.32%, #FEFBFF 49.94%, #EDA7FD 71.83%, #E474FC 83.75%, #F174FD 100%);"
+    },{
+      dbz: ">65",
+      description: "Extremely heavy precipitation including water-coated hail",
+      gradient: "180deg, #F174FD 0%, #F875FF 30.26%, #AA0BFA 34%, #5B06D3 98.5%"
+    }]
+    %}
+
+    <div class="padding-x-2 padding-bottom-1 margin-top-4 wx-radar-legend">
+      <h4 class="visual-h3 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">{{ "Radar legend" | t }}</h4>
+
+      <table class="usa-table usa-table--borderless margin-top-0 wx-radar-legend" tabindex="0">
```

Comment:
I was wrong in my original comment, we should leave that radar description as is, but perhaps include a SR only caption here with the table description:
"Legend which describes the intensity levels that correspond to color and dBz values"